CS2113 IP/src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,306 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Duke {
+    final private static String GOODBYE_COMMENT = "Bye. Hope to see you again soon!";
+    final private static String ERROR_MARK_TASK = "Please do not leave your task number empty :-(";
+    final private static String ERROR_UNKNOWN_INPUT = ":-( OOPS!!! I'm sorry, but I don't know what that means :-(";
+    final private static String ERROR_EMPTY_TODO_DESCRIPTION = "Please do not leave your todo description empty :-(";
+    final private static String ERROR_EMPTY_DEADLINE_DESCRIPTION = "Please do not leave your deadline description empty :-(";
+    final private static String ERROR_EMPTY_EVENT_DESCRIPTION = "Please do not leave your event description empty :-(";
+    final private static String ERROR_EMPTY_DELETE_DESCRIPTION = "Please do not leave your delete task number empty :-(";
+    final private static String ERROR_WRONG_HANDLE_TODO_DESCRIPTION = "Check for missing fields in your description!";
+    final private static String ERROR_WRONG_HANDLE_EVENT_DESCRIPTION = "Include /at handler and insert date of event!";
+    final private static String ERROR_WRONG_HANDLE_DEADLINE_DESCRIPTION = "Include /by handler and insert deadline!";
+    final private static ArrayList<Task> tasks = new ArrayList<Task>();
+    final private static String filePath = getFilePath();
+
+    public static void Greet() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println(logo);
+        String horizontalLine = "________________________";
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        System.out.println(horizontalLine);
+    }
+
+    public static void listOperations() throws IOException {
+
+        Scanner sc = new Scanner(System.in);
+        String horizontalLine = "________________________";
+
+        boolean isBye;
+        boolean isList;
+        boolean isDone;
+        boolean isTodo;
+        boolean isDeadline;
+        boolean isEvent;
+        boolean isDelete;
+
+        do {
+            String userInput = sc.nextLine();
+
+            isBye = userInput.equals("bye");
+            isList = userInput.equals("list");
+            isDone = userInput.startsWith("done");
+            isTodo = userInput.startsWith("todo");
+            isDeadline = userInput.startsWith("deadline");
+            isEvent = userInput.startsWith("event");
+            isDelete = userInput.startsWith("delete");
+            System.out.println(horizontalLine);
+
+            if (isBye) {
```

Comment:
Well done, do consider using a switch statement in place of "else if" statements as it is slightly preferred (no penalty).